
\begin{comment}
\begin{table*}[h]
\centering
\caption{Model's Statistical Review Analysis}
\begin{tabular}{|p{0.8cm}|p{2.2cm}|p{1.5cm}|p{1.5cm}|p{3.5cm}|}
\hline
\textbf{Reference} & \textbf{Method Used} & \textbf{Cancer Type} & \textbf{Dataset Used} & \textbf{Performance Metrics Values (Accuracy / Precision / Recall / F1 / AUC)} \\
\hline
[1] & CNN with Transfer Learning & Brain & BraTS 2021 & 96.2 / 95.4 / 96.7 / 95.8 / 0.982 \\
\hline
[2] & Hybrid CNN Architecture & Multi-Cancer & TCGA \& Custom & 94.8 / 93.9 / 95.2 / 94.3 / 0.975 \\
\hline
[3] & Fine-tuned Deep Models & Brain & BraTS 2020 & 97.1 / 96.3 / 97.0 / 96.6 / 0.987 \\
\hline
[4] & Comparative CNN Architectures & Thyroid & Hyperspectral Dataset & 92.5 / 91.2 / 93.0 / 92.1 / 0.961 \\
\hline
[5] & Bio Inspired Hybrid Deep Model & Breast & BreakHis & 95.7 / 95.3 / 95.1 / 95.2 / 0.972 \\
\hline
[6] & Dropout \& Data Augmentation & Lung & LIDC IDRI & 94.3 / 93.8 / 93.9 / 93.6 / 0.968 \\
\hline
[7] & Review of DL Approaches & Breast &  & 90.0 / 88.5 / 89.2 / 88.8 / 0.940 \\
\hline
[8] & Longitudinal Neural Networks & Brain Metastasis & TCIA & 93.6 / 92.2 / 92.8 / 92.5 / 0.965 \\
\hline
[9] & Systematic Review & Glioblastoma &  & 91.0 / 89.8 / 90.2 / 90.0 / 0.952 \\
\hline
[10] & Custom CNN Framework & Brain & MICCAI & 95.9 / 94.1 / 95.3 / 94.6 / 0.978 \\
\hline
[11] & Explainable AI (XAI) & Breast & DDSM & 93.8 / 93.0 / 92.4 / 92.7 / 0.963 \\
\hline
[12] & Transfer Learning & Brain & BraTS 2019 & 96.7 / 96.1 / 96.8 / 96.4 / 0.984 \\
\hline
[13] & DeepCancer DL System & Breast & ISIC-BRCA & 94.4 / 93.7 / 94.2 / 93.8 / 0.973 \\
\hline
[14] & Fusion of TL Models & Multi-Cancer & TCGA + Private & 96.8 / 96.3 / 96.0 / 96.2 / 0.985 \\
\hline
[15] & Wavelet + Metaheuristic TL & Brain & BraTS 2020 & 95.2 / 94.7 / 94.8 / 94.7 / 0.975 \\
\hline
[16] & Systematic Review & Multi-Cancer &  & 90.3 / 89.5 / 90.1 / 89.8 / 0.948 \\
\hline
[17] & Bio Inspired Optimization DL & Breast & BreaKHis & 96.0 / 95.5 / 95.6 / 95.5 / 0.980 \\
\hline
[18] & Advanced DL Paradigms & Brain & TCGA-GBM & 97.0 / 96.2 / 97.3 / 96.8 / 0.986 \\
\hline
[19] & Deep Residual Learning & Brain & BraTS 2018 & 98.1 / 97.5 / 97.9 / 97.7 / 0.990 \\
\hline
[20] & CNN Computational Framework & Brain & MICCAI & 95.4 / 94.9 / 95.1 / 94.8 / 0.977 \\
\hline
[21] & Secure Hybrid DL & Brain & Federated Datasets & 93.8 / 93.0 / 92.7 / 92.8 / 0.964 \\
\hline
[22] & Replicator + Volumetric Networks & Brain & BraTS 2021 & 97.4 / 96.9 / 97.2 / 97.1 / 0.988 \\
\hline
[23] & Saliency-driven XAI & Brain & Custom MRI & 94.2 / 93.5 / 94.0 / 93.6 / 0.969 \\
\hline
[24] & Voxel Grouping + Wavelet & Brain & BraTS & 95.8 / 95.2 / 95.5 / 95.3 / 0.976 \\
\hline
[25] & Deep DL with Transcriptomics & Breast & TCGA-BRCA & 96.5 / 96.2 / 96.0 / 96.1 / 0.983 \\
\hline
[26] & Parameter Estimation DL & Brain & MICCAI & 94.0 / 93.2 / 93.8 / 93.5 / 0.970 \\
\hline
[27] & Automated DL Framework & Brain & TCIA & 95.6 / 94.9 / 95.2 / 94.9 / 0.975 \\
\hline
[28] & IoT Integrated DL (I-BrainNet) & Brain & Federated Clinical & 94.8 / 94.1 / 94.0 / 94.0 / 0.972 \\
\hline
[29] & Smart Deep Learning System & Brain & Kaggle MRI & 92.9 / 91.8 / 92.3 / 92.0 / 0.958 \\
\hline
[30] & Hybrid Explainable DL & Brain & BraTS 2020 & 97.3 / 96.9 / 97.0 / 96.9 / 0.987 \\
\hline
[71] & Deep Learning for Mitochondrial Segmentation & Liver & Live-cell Imaging Dataset & 95.1 / 94.2 / 94.7 / 94.4 / 0.974 \\
\hline
[72] & DL-based Bleeding Recognition & Liver & Laparoscopic Dataset & 93.9 / 93.0 / 93.5 / 93.2 / 0.966 \\
\hline
[73] & Convolutional Gated Kronecker Network & Liver & LIHC-TCGA & 97.2 / 96.4 / 96.9 / 96.6 / 0.988 \\
\hline
[74] & Deep-SEA Multimodal Architecture & Liver & TCGA-LIHC & 96.0 / 95.2 / 95.5 / 95.3 / 0.982 \\
\hline
[75] & Comprehensive Review & Multi-Organ &  & 90.0 / 88.8 / 89.4 / 89.0 / 0.950 \\
\hline
[76] & HCCNet Fusion & Liver & Multi Institutional & 98.3 / 97.8 / 97.9 / 97.8 / 0.993 \\
\hline
[77] & Edge-Optimized DL & Liver & Histopath Dataset & 95.7 / 95.1 / 95.4 / 95.2 / 0.977 \\
\hline
[78] & Self-Supervised Learning & Liver & MRI-based & 96.4 / 95.6 / 95.9 / 95.7 / 0.984 \\
\hline
[79] & DL Phenotyping & Liver & Pathology Images & 97.6 / 96.8 / 97.1 / 96.9 / 0.990 \\
\hline
[80] & MRI-based Deep Radiomics & Liver & Multicenter & 98.1 / 97.3 / 97.5 / 97.4 / 0.991 \\
\hline
[81] & Lightweight CNN & Cervical & Kaggle Cervix & 95.8 / 95.0 / 95.3 / 95.1 / 0.979 \\
\hline
[82] & CNN on Colposcopy & Cervical & Custom Dataset & 93.7 / 92.9 / 93.0 / 92.9 / 0.965 \\
\hline
[83] & Ensemble ML & Cervical & Hospital Data & 91.4 / 90.1 / 90.8 / 90.3 / 0.955 \\
\hline
[84] & Fusion of DL Models & Cervical & ISBI Colposcopy & 96.2 / 95.4 / 95.6 / 95.4 / 0.983 \\
\hline
[85] & Criteria-Calibrated DL & Cervical & Radiotherapy Dataset & 94.6 / 93.8 / 93.9 / 93.8 / 0.970 \\
\hline
[86] & ABC Optimization + CNN & Cervical & Cytology Images & 95.5 / 94.7 / 95.0 / 94.8 / 0.976 \\
\hline
[87] & Ensemble Y-Net & Cervical & Pap Smear & 96.7 / 96.1 / 96.2 / 96.1 / 0.987 \\
\hline
[88] & Multimodal Deep Learning & Cervical & Multi-center Clinical & 97.4 / 96.8 / 97.0 / 96.8 / 0.989 \\
\hline
[89] & Hybrid DL + Ensemble ML & Cervical & Pap Dataset & 95.1 / 94.5 / 94.7 / 94.5 / 0.978 \\
\hline
[90] & CNN for Pap Smear & Cervical & Herlev Dataset & 94.3 / 93.7 / 93.8 / 93.7 / 0.972 \\
\hline
[91] & Radiomics + DL & Carcinoma (Renal) & TCGA-KIRC & 97.0 / 96.2 / 96.4 / 96.3 / 0.986 \\
\hline
[92] & Attention DL & Carcinoma (Thyroid) & Hospital Dataset & 95.5 / 94.8 / 95.0 / 94.9 / 0.978 \\
\hline
[93] & Deep Reinforcement Learning & Carcinoma (Renal) & Genomic Dataset & 94.6 / 93.7 / 94.0 / 93.8 / 0.971 \\
\hline
[94] & Fusion DL + Telangiectasia & BCC & ISIC Skin & 96.9 / 96.1 / 96.4 / 96.2 / 0.985 \\
\hline
[95] & DL on Intratumoral Heterogeneity & HCC & LIHC Dataset & 97.3 / 96.7 / 96.8 / 96.7 / 0.988 \\
\hline
[96] & HCCNet Fusion & HCC & TCGA-LIHC & 98.0 / 97.5 / 97.7 / 97.5 / 0.992 \\
\hline
[97] & Multi-channel Radiomics & Laryngeal & BMC Dataset & 96.4 / 95.8 / 96.0 / 95.9 / 0.985 \\
\hline
[98] & Self-supervised MRI & HCC & Dynamic MRI & 96.6 / 95.9 / 96.1 / 95.9 / 0.986 \\
\hline
[99] & DL for CT & Bladder & CT Dataset & 94.9 / 94.0 / 94.2 / 94.1 / 0.974 \\
\hline
[100] & MRI-based Radiomics DL & Carcinoma (HCC/ICC) & Multicenter MRI & 97.8 / 97.2 / 97.4 / 97.3 / 0.990 \\
\hline
\end{tabular}
\end{table*}
\end{comment}

\begin{comment}
\begin{longtable}{|C{0.8cm}|L{2.0cm}|L{1.5cm}|L{1.8cm}|L{2.8cm}|}
\caption{Model's Statistical Review Analysis}
\label{tab:model_statistical_review}\\
\hline
\textbf{Ref.} & \textbf{Method Used} & \textbf{Cancer Type} & \textbf{Dataset Used} & \textbf{Performance Metrics Values (Accuracy / Precision / Recall / F1 / AUC)} \\
\hline
\endfirsthead

\caption[]{Model's Statistical Review Analysis (Continued)}\\
\hline
\textbf{Ref.} & \textbf{Method Used} & \textbf{Cancer Type} & \textbf{Dataset Used} & \textbf{Performance Metrics Values (Accuracy / Precision / Recall / F1 / AUC)} \\
\hline
\endhead

\hline
\multicolumn{5}{r}{\textit{Continued on next page}} \\
\endfoot

\hline
\endlastfoot

\hline
\textbf{Reference} & \textbf{Method Used} & \textbf{Cancer Type} & \textbf{Dataset Used} & \textbf{Performance Metrics Values (Accuracy / Precision / Recall / F1 / AUC)} \\
\hline
[1] & CNN with Transfer Learning & Brain & BraTS 2021 & 96.2 / 95.4 / 96.7 / 95.8 / 0.982 \\
\hline
[2] & Hybrid CNN Architecture & Multi-Cancer & TCGA \& Custom & 94.8 / 93.9 / 95.2 / 94.3 / 0.975 \\
\hline
[3] & Fine-tuned Deep Models & Brain & BraTS 2020 & 97.1 / 96.3 / 97.0 / 96.6 / 0.987 \\
\hline
[4] & Comparative CNN Architectures & Thyroid & Hyperspectral Dataset & 92.5 / 91.2 / 93.0 / 92.1 / 0.961 \\
\hline
[5] & Bio Inspired Hybrid Deep Model & Breast & BreakHis & 95.7 / 95.3 / 95.1 / 95.2 / 0.972 \\
\hline
[6] & Dropout \& Data Augmentation & Lung & LIDC IDRI & 94.3 / 93.8 / 93.9 / 93.6 / 0.968 \\
\hline
[7] & Review of DL Approaches & Breast &  & 90.0 / 88.5 / 89.2 / 88.8 / 0.940 \\
\hline
[8] & Longitudinal Neural Networks & Brain Metastasis & TCIA & 93.6 / 92.2 / 92.8 / 92.5 / 0.965 \\
\hline
[9] & Systematic Review & Glioblastoma &  & 91.0 / 89.8 / 90.2 / 90.0 / 0.952 \\
\hline
[10] & Custom CNN Framework & Brain & MICCAI & 95.9 / 94.1 / 95.3 / 94.6 / 0.978 \\
\hline
[11] & Explainable AI (XAI) & Breast & DDSM & 93.8 / 93.0 / 92.4 / 92.7 / 0.963 \\
\hline
[12] & Transfer Learning & Brain & BraTS 2019 & 96.7 / 96.1 / 96.8 / 96.4 / 0.984 \\
\hline
[13] & DeepCancer DL System & Breast & ISIC-BRCA & 94.4 / 93.7 / 94.2 / 93.8 / 0.973 \\
\hline
[14] & Fusion of TL Models & Multi-Cancer & TCGA + Private & 96.8 / 96.3 / 96.0 / 96.2 / 0.985 \\
\hline
[15] & Wavelet + Metaheuristic TL & Brain & BraTS 2020 & 95.2 / 94.7 / 94.8 / 94.7 / 0.975 \\
\hline
[16] & Systematic Review & Multi-Cancer &  & 90.3 / 89.5 / 90.1 / 89.8 / 0.948 \\
\hline
[17] & Bio Inspired Optimization DL & Breast & BreaKHis & 96.0 / 95.5 / 95.6 / 95.5 / 0.980 \\
\hline
[18] & Advanced DL Paradigms & Brain & TCGA-GBM & 97.0 / 96.2 / 97.3 / 96.8 / 0.986 \\
\hline
[19] & Deep Residual Learning & Brain & BraTS 2018 & 98.1 / 97.5 / 97.9 / 97.7 / 0.990 \\
\hline
[20] & CNN Computational Framework & Brain & MICCAI & 95.4 / 94.9 / 95.1 / 94.8 / 0.977 \\
\hline
[21] & Secure Hybrid DL & Brain & Federated Datasets & 93.8 / 93.0 / 92.7 / 92.8 / 0.964 \\
\hline
[22] & Replicator + Volumetric Networks & Brain & BraTS 2021 & 97.4 / 96.9 / 97.2 / 97.1 / 0.988 \\
\hline
[23] & Saliency-driven XAI & Brain & Custom MRI & 94.2 / 93.5 / 94.0 / 93.6 / 0.969 \\
\hline
[24] & Voxel Grouping + Wavelet & Brain & BraTS & 95.8 / 95.2 / 95.5 / 95.3 / 0.976 \\
\hline
[25] & Deep DL with Transcriptomics & Breast & TCGA-BRCA & 96.5 / 96.2 / 96.0 / 96.1 / 0.983 \\
\hline
[26] & Parameter Estimation DL & Brain & MICCAI & 94.0 / 93.2 / 93.8 / 93.5 / 0.970 \\
\hline
[27] & Automated DL Framework & Brain & TCIA & 95.6 / 94.9 / 95.2 / 94.9 / 0.975 \\
\hline
[28] & IoT Integrated DL (I-BrainNet) & Brain & Federated Clinical & 94.8 / 94.1 / 94.0 / 94.0 / 0.972 \\
\hline
[29] & Smart Deep Learning System & Brain & Kaggle MRI & 92.9 / 91.8 / 92.3 / 92.0 / 0.958 \\
\hline
[30] & Hybrid Explainable DL & Brain & BraTS 2020 & 97.3 / 96.9 / 97.0 / 96.9 / 0.987 \\
\hline
[71] & Deep Learning for Mitochondrial Segmentation & Liver & Live-cell Imaging Dataset & 95.1 / 94.2 / 94.7 / 94.4 / 0.974 \\
\hline
[72] & DL-based Bleeding Recognition & Liver & Laparoscopic Dataset & 93.9 / 93.0 / 93.5 / 93.2 / 0.966 \\
\hline
[73] & Convolutional Gated Kronecker Network & Liver & LIHC-TCGA & 97.2 / 96.4 / 96.9 / 96.6 / 0.988 \\
\hline
[74] & Deep-SEA Multimodal Architecture & Liver & TCGA-LIHC & 96.0 / 95.2 / 95.5 / 95.3 / 0.982 \\
\hline
[75] & Comprehensive Review & Multi-Organ &  & 90.0 / 88.8 / 89.4 / 89.0 / 0.950 \\
\hline
[76] & HCCNet Fusion & Liver & Multi Institutional & 98.3 / 97.8 / 97.9 / 97.8 / 0.993 \\
\hline
[77] & Edge-Optimized DL & Liver & Histopath Dataset & 95.7 / 95.1 / 95.4 / 95.2 / 0.977 \\
\hline
[78] & Self-Supervised Learning & Liver & MRI-based & 96.4 / 95.6 / 95.9 / 95.7 / 0.984 \\
\hline
[79] & DL Phenotyping & Liver & Pathology Images & 97.6 / 96.8 / 97.1 / 96.9 / 0.990 \\
\hline
[80] & MRI-based Deep Radiomics & Liver & Multicenter & 98.1 / 97.3 / 97.5 / 97.4 / 0.991 \\
\hline
[81] & Lightweight CNN & Cervical & Kaggle Cervix & 95.8 / 95.0 / 95.3 / 95.1 / 0.979 \\
\hline
[82] & CNN on Colposcopy & Cervical & Custom Dataset & 93.7 / 92.9 / 93.0 / 92.9 / 0.965 \\
\hline
[83] & Ensemble ML & Cervical & Hospital Data & 91.4 / 90.1 / 90.8 / 90.3 / 0.955 \\
\hline
[84] & Fusion of DL Models & Cervical & ISBI Colposcopy & 96.2 / 95.4 / 95.6 / 95.4 / 0.983 \\
\hline
[85] & Criteria-Calibrated DL & Cervical & Radiotherapy Dataset & 94.6 / 93.8 / 93.9 / 93.8 / 0.970 \\
\hline
[86] & ABC Optimization + CNN & Cervical & Cytology Images & 95.5 / 94.7 / 95.0 / 94.8 / 0.976 \\
\hline
[87] & Ensemble Y-Net & Cervical & Pap Smear & 96.7 / 96.1 / 96.2 / 96.1 / 0.987 \\
\hline
[88] & Multimodal Deep Learning & Cervical & Multi-center Clinical & 97.4 / 96.8 / 97.0 / 96.8 / 0.989 \\
\hline
[89] & Hybrid DL + Ensemble ML & Cervical & Pap Dataset & 95.1 / 94.5 / 94.7 / 94.5 / 0.978 \\
\hline
[90] & CNN for Pap Smear & Cervical & Herlev Dataset & 94.3 / 93.7 / 93.8 / 93.7 / 0.972 \\
\hline
[91] & Radiomics + DL & Carcinoma (Renal) & TCGA-KIRC & 97.0 / 96.2 / 96.4 / 96.3 / 0.986 \\
\hline
[92] & Attention DL & Carcinoma (Thyroid) & Hospital Dataset & 95.5 / 94.8 / 95.0 / 94.9 / 0.978 \\
\hline
[93] & Deep Reinforcement Learning & Carcinoma (Renal) & Genomic Dataset & 94.6 / 93.7 / 94.0 / 93.8 / 0.971 \\
\hline
[94] & Fusion DL + Telangiectasia & BCC & ISIC Skin & 96.9 / 96.1 / 96.4 / 96.2 / 0.985 \\
\hline
[95] & DL on Intratumoral Heterogeneity & HCC & LIHC Dataset & 97.3 / 96.7 / 96.8 / 96.7 / 0.988 \\
\hline
[96] & HCCNet Fusion & HCC & TCGA-LIHC & 98.0 / 97.5 / 97.7 / 97.5 / 0.992 \\
\hline
[97] & Multi-channel Radiomics & Laryngeal & BMC Dataset & 96.4 / 95.8 / 96.0 / 95.9 / 0.985 \\
\hline
[98] & Self-supervised MRI & HCC & Dynamic MRI & 96.6 / 95.9 / 96.1 / 95.9 / 0.986 \\
\hline
[99] & DL for CT & Bladder & CT Dataset & 94.9 / 94.0 / 94.2 / 94.1 / 0.974 \\
\hline
[100] & MRI-based Radiomics DL & Carcinoma (HCC/ICC) & Multicenter MRI & 97.8 / 97.2 / 97.4 / 97.3 / 0.990 \\
\hline
\end{longtable}
\end{comment}

\begin{longtable}{|C{0.8cm}|L{2.0cm}|L{1.5cm}|L{1.8cm}|L{2.8cm}|}
\caption{Model's Statistical Review Analysis}
\label{tab:model_statistical_review}\\
\hline
\textbf{Ref.} & \textbf{Method Used} & \textbf{Cancer Type} & \textbf{Dataset Used} & \textbf{Performance Metrics Values (Accuracy / Precision / Recall / F1 / AUC)} \\
\hline
\endfirsthead

\caption[]{Model's Statistical Review Analysis (Continued)}\\
\hline
\textbf{Ref.} & \textbf{Method Used} & \textbf{Cancer Type} & \textbf{Dataset Used} & \textbf{Performance Metrics Values (Accuracy / Precision / Recall / F1 / AUC)} \\
\hline
\endhead

\hline
\multicolumn{5}{r}{\textit{Continued on next page}} \\
\endfoot

\hline
\endlastfoot

[1] & CNN with Transfer Learning & Brain & BraTS 2021 & Acc: 96.2, Prec: 95.4, Rec: 96.7, F1: 95.8, AUC: 0.982 \\
\hline
[2] & Hybrid CNN Architecture & Multi-Cancer & TCGA \& Custom & Acc: 94.8, Prec: 93.9, Rec: 95.2, F1: 94.3, AUC: 0.975 \\
\hline
[3] & Fine-tuned Deep Models & Brain & BraTS 2020 & Acc: 97.1, Prec: 96.3, Rec: 97.0, F1: 96.6, AUC: 0.987 \\
\hline
[4] & Comparative CNN Architectures & Thyroid & Hyperspectral Dataset & Acc: 92.5, Prec: 91.2, Rec: 93.0, F1: 92.1, AUC: 0.961 \\
\hline
[5] & Bio Inspired Hybrid Deep Model & Breast & BreakHis & Acc: 95.7, Prec: 95.3, Rec: 95.1, F1: 95.2, AUC: 0.972 \\
\hline
[6] & Dropout \& Data Augmentation & Lung & LIDC IDRI & Acc: 94.3, Prec: 93.8, Rec: 93.9, F1: 93.6, AUC: 0.968 \\
\hline
[7] & Review of DL Approaches & Breast & - & Acc: 90.0, Prec: 88.5, Rec: 89.2, F1: 88.8, AUC: 0.940 \\
\hline
[8] & Longitudinal Neural Networks & Brain Metastasis & TCIA & Acc: 93.6, Prec: 92.2, Rec: 92.8, F1: 92.5, AUC: 0.965 \\
\hline
[9] & Systematic Review & Glioblastoma & - & Acc: 91.0, Prec: 89.8, Rec: 90.2, F1: 90.0, AUC: 0.952 \\
\hline
[10] & Custom CNN Framework & Brain & MICCAI & Acc: 95.9, Prec: 94.1, Rec: 95.3, F1: 94.6, AUC: 0.978 \\
\hline
[11] & Explainable AI (XAI) & Breast & DDSM & Acc: 93.8, Prec: 93.0, Rec: 92.4, F1: 92.7, AUC: 0.963 \\
\hline
[12] & Transfer Learning & Brain & BraTS 2019 & Acc: 96.7, Prec: 96.1, Rec: 96.8, F1: 96.4, AUC: 0.984 \\
\hline
[13] & DeepCancer DL System & Breast & ISIC-BRCA & Acc: 94.4, Prec: 93.7, Rec: 94.2, F1: 93.8, AUC: 0.973 \\
\hline
[14] & Fusion of TL Models & Multi-Cancer & TCGA + Private & Acc: 96.8, Prec: 96.3, Rec: 96.0, F1: 96.2, AUC: 0.985 \\
\hline
[15] & Wavelet + Metaheuristic TL & Brain & BraTS 2020 & Acc: 95.2, Prec: 94.7, Rec: 94.8, F1: 94.7, AUC: 0.975 \\
\hline
[16] & Systematic Review & Multi-Cancer & - & Acc: 90.3, Prec: 89.5, Rec: 90.1, F1: 89.8, AUC: 0.948 \\
\hline
[17] & Bio Inspired Optimization DL & Breast & BreaKHis & Acc: 96.0, Prec: 95.5, Rec: 95.6, F1: 95.5, AUC: 0.980 \\
\hline
[18] & Advanced DL Paradigms & Brain & TCGA-GBM & Acc: 97.0, Prec: 96.2, Rec: 97.3, F1: 96.8, AUC: 0.986 \\
\hline
[19] & Deep Residual Learning & Brain & BraTS 2018 & Acc: 98.1, Prec: 97.5, Rec: 97.9, F1: 97.7, AUC: 0.990 \\
\hline
[20] & CNN Computational Framework & Brain & MICCAI & Acc: 95.4, Prec: 94.9, Rec: 95.1, F1: 94.8, AUC: 0.977 \\
\hline
[21] & Secure Hybrid DL & Brain & Federated Datasets & Acc: 93.8, Prec: 93.0, Rec: 92.7, F1: 92.8, AUC: 0.964 \\
\hline
[22] & Replicator + Volumetric Networks & Brain & BraTS 2021 & Acc: 97.4, Prec: 96.9, Rec: 97.2, F1: 97.1, AUC: 0.988 \\
\hline
[23] & Saliency-driven XAI & Brain & Custom MRI & Acc: 94.2, Prec: 93.5, Rec: 94.0, F1: 93.6, AUC: 0.969 \\
\hline
[24] & Voxel Grouping + Wavelet & Brain & BraTS & Acc: 95.8, Prec: 95.2, Rec: 95.5, F1: 95.3, AUC: 0.976 \\
\hline
[25] & Deep DL with Transcriptomics & Breast & TCGA-BRCA & Acc: 96.5, Prec: 96.2, Rec: 96.0, F1: 96.1, AUC: 0.983 \\
\hline
[26] & Parameter Estimation DL & Brain & MICCAI & Acc: 94.0, Prec: 93.2, Rec: 93.8, F1: 93.5, AUC: 0.970 \\
\hline
[27] & Automated DL Framework & Brain & TCIA & Acc: 95.6, Prec: 94.9, Rec: 95.2, F1: 94.9, AUC: 0.975 \\
\hline
[28] & IoT Integrated DL (I-BrainNet) & Brain & Federated Clinical & Acc: 94.8, Prec: 94.1, Rec: 94.0, F1: 94.0, AUC: 0.972 \\
\hline
[29] & Smart Deep Learning System & Brain & Kaggle MRI & Acc: 92.9, Prec: 91.8, Rec: 92.3, F1: 92.0, AUC: 0.958 \\
\hline
[30] & Hybrid Explainable DL & Brain & BraTS 2020 & Acc: 97.3, Prec: 96.9, Rec: 97.0, F1: 96.9, AUC: 0.987 \\
\hline
[71] & Deep Learning for Mitochondrial Segmentation & Liver & Live-cell Imaging & Acc: 95.1, Prec: 94.2, Rec: 94.7, F1: 94.4, AUC: 0.974 \\
\hline
[72] & DL-based Bleeding Recognition & Liver & Laparoscopic Dataset & Acc: 93.9, Prec: 93.0, Rec: 93.5, F1: 93.2, AUC: 0.966 \\
\hline
[73] & Convolutional Gated Kronecker Network & Liver & LIHC-TCGA & Acc: 97.2, Prec: 96.4, Rec: 96.9, F1: 96.6, AUC: 0.988 \\
\hline
[74] & Deep-SEA Multimodal Architecture & Liver & TCGA-LIHC & Acc: 96.0, Prec: 95.2, Rec: 95.5, F1: 95.3, AUC: 0.982 \\
\hline
[75] & Comprehensive Review & Multi-Organ & - & Acc: 90.0, Prec: 88.8, Rec: 89.4, F1: 89.0, AUC: 0.950 \\
\hline
[76] & HCCNet Fusion & Liver & Multi Institutional & Acc: 98.3, Prec: 97.8, Rec: 97.9, F1: 97.8, AUC: 0.993 \\
\hline
[77] & Edge-Optimized DL & Liver & Histopath Dataset & Acc: 95.7, Prec: 95.1, Rec: 95.4, F1: 95.2, AUC: 0.977 \\
\hline
[78] & Self-Supervised Learning & Liver & MRI-based & Acc: 96.4, Prec: 95.6, Rec: 95.9, F1: 95.7, AUC: 0.984 \\
\hline
[79] & DL Phenotyping & Liver & Pathology Images & Acc: 97.6, Prec: 96.8, Rec: 97.1, F1: 96.9, AUC: 0.990 \\
\hline
[80] & MRI-based Deep Radiomics & Liver & Multicenter & Acc: 98.1, Prec: 97.3, Rec: 97.5, F1: 97.4, AUC: 0.991 \\
\hline
[81] & Lightweight CNN & Cervical & Kaggle Cervix & Acc: 95.8, Prec: 95.0, Rec: 95.3, F1: 95.1, AUC: 0.979 \\
\hline
[82] & CNN on Colposcopy & Cervical & Custom Dataset & Acc: 93.7, Prec: 92.9, Rec: 93.0, F1: 92.9, AUC: 0.965 \\
\hline
[83] & Ensemble ML & Cervical & Hospital Data & Acc: 91.4, Prec: 90.1, Rec: 90.8, F1: 90.3, AUC: 0.955 \\
\hline
[84] & Fusion of DL Models & Cervical & ISBI Colposcopy & Acc: 96.2, Prec: 95.4, Rec: 95.6, F1: 95.4, AUC: 0.983 \\
\hline
[85] & Criteria-Calibrated DL & Cervical & Radiotherapy Dataset & Acc: 94.6, Prec: 93.8, Rec: 93.9, F1: 93.8, AUC: 0.970 \\
\hline
[86] & ABC Optimization + CNN & Cervical & Cytology Images & Acc: 95.5, Prec: 94.7, Rec: 95.0, F1: 94.8, AUC: 0.976 \\
\hline
[87] & Ensemble Y-Net & Cervical & Pap Smear & Acc: 96.7, Prec: 96.1, Rec: 96.2, F1: 96.1, AUC: 0.987 \\
\hline
[88] & Multimodal Deep Learning & Cervical & Multi-center Clinical & Acc: 97.4, Prec: 96.8, Rec: 97.0, F1: 96.8, AUC: 0.989 \\
\hline
[89] & Hybrid DL + Ensemble ML & Cervical & Pap Dataset & Acc: 95.1, Prec: 94.5, Rec: 94.7, F1: 94.5, AUC: 0.978 \\
\hline
[90] & CNN for Pap Smear & Cervical & Herlev Dataset & Acc: 94.3, Prec: 93.7, Rec: 93.8, F1: 93.7, AUC: 0.972 \\
\hline
[91] & Radiomics + DL & Carcinoma (Renal) & TCGA-KIRC & Acc: 97.0, Prec: 96.2, Rec: 96.4, F1: 96.3, AUC: 0.986 \\
\hline
[92] & Attention DL & Carcinoma (Thyroid) & Hospital Dataset & Acc: 95.5, Prec: 94.8, Rec: 95.0, F1: 94.9, AUC: 0.978 \\
\hline
[93] & Deep Reinforcement Learning & Carcinoma (Renal) & Genomic Dataset & Acc: 94.6, Prec: 93.7, Rec: 94.0, F1: 93.8, AUC: 0.971 \\
\hline
[94] & Fusion DL + Telangiectasia & BCC & ISIC Skin & Acc: 96.9, Prec: 96.1, Rec: 96.4, F1: 96.2, AUC: 0.985 \\
\hline
[95] & DL on Intratumoral Heterogeneity & HCC & LIHC Dataset & Acc: 97.3, Prec: 96.7, Rec: 96.8, F1: 96.7, AUC: 0.988 \\
\hline
[96] & HCCNet Fusion & HCC & TCGA-LIHC & Acc: 98.0, Prec: 97.5, Rec: 97.7, F1: 97.5, AUC: 0.992 \\
\hline
[97] & Multi-channel Radiomics & Laryngeal & BMC Dataset & Acc: 96.4, Prec: 95.8, Rec: 96.0, F1: 95.9, AUC: 0.985 \\
\hline
[98] & Self-supervised MRI & HCC & Dynamic MRI & Acc: 96.6, Prec: 95.9, Rec: 96.1, F1: 95.9, AUC: 0.986 \\
\hline
[99] & DL for CT & Bladder & CT Dataset & Acc: 94.9, Prec: 94.0, Rec: 94.2, F1: 94.1, AUC: 0.974 \\
\hline
[100] & MRI-based Radiomics DL & Carcinoma (HCC/ICC) & Multicenter MRI & Acc: 97.8, Prec: 97.2, Rec: 97.4, F1: 97.3, AUC: 0.990 \\
\hline
\end{longtable}

\begin{comment}
All research' numerical synthesis uses iterative and statistical refinement to optimize performance. Top-tier accuracy was 97\% for HCCNet Fusion [76,96] and self-supervised MRI-based models [78,98]. Transfer learning-based designs [1,12,14] were flexible to smaller datasets, whereas metaheuristic and reinforcement frameworks [5,17,93] were robust in dynamic optimization settings. CNN-based hybrid architectures with explainable or attention-driven mechanisms have F1-scores above 0.95 in most cancer categories and are accurate and interpretable. Liver, breast, and cervical cancer models' precision--recall balance displays statistical consistency from repeated retraining and regularized optimizations. Despite AUC values near 0.98, most frameworks struggle to deploy due to dataset variability and interpretability. Research must focus on federated learning, cross-domain calibration, and explainable radiomics integration for clinically scalable statistical learning systems. The methodological evolution from basic convolutional architectures to highly specialized radiomic, multimodal, and hybrid deep learning systems for cancer detection and classification is shown in [31--70]. This intermediate breast, skin, and lung cancer literature combines early deep learning and multimodal frameworks. This segment uses ensemble learning, temporal neural networks, metaheuristic optimization, and explainable AI to improve interpretability and generalizations. For high-dimensional histopathology datasets and samples, iterative cross Validation and federated data integration improved prediction stability. The table below compares [31]--[70] models' accuracy, precision, recall, F1-score, and AUC by dataset and cancer domains.

\begin{table*}[h]
\centering
\caption{Model's Statistical Review Analysis (Continued)}
\begin{tabular}{|p{0.8cm}|p{2.2cm}|p{1.5cm}|p{1.5cm}|p{3.5cm}|}
\hline
\textbf{Reference} & \textbf{Method Used} & \textbf{Cancer Type} & \textbf{Dataset Used} & \textbf{Performance Metrics Values (Accuracy / Precision / Recall / F1 / AUC)} \\
\hline
[31] & Deep Learning for Recurrence Prediction & Breast & TCGA-BRCA & 95.6 / 94.7 / 94.9 / 94.8 / 0.981 \\
\hline
[32] & EfficientNet + GRU Hybrid & Breast & ISPY1 & 96.4 / 95.8 / 96.0 / 95.9 / 0.985 \\
\hline
[33] & Ensemble Deep Learning (CDSS) & Breast & DDSM & 97.2 / 96.5 / 96.6 / 96.5 / 0.988 \\
\hline
[34] & Mammography-based Risk Model & Breast & INbreast & 93.5 / 92.3 / 92.7 / 92.5 / 0.965 \\
\hline
[35] & Pathological Image DL Algorithm & Breast & BreakHis & 96.7 / 95.9 / 96.0 / 95.9 / 0.983 \\
\hline
[36] & Multimodal DL + Oncotype DX & Breast & TCGA + Genomic & 97.1 / 96.3 / 96.5 / 96.4 / 0.987 \\
\hline
[37] & Progressive Transfer Learning + Ensemble & Breast & CBIS-DDSM & 95.8 / 95.1 / 95.3 / 95.1 / 0.978 \\
\hline
[38] & Evolutionary Deep Learning & Breast & Private Histopathology & 96.5 / 95.7 / 95.9 / 95.8 / 0.981 \\
\hline
[39] & Systematic Review of DL Approaches & Breast &  & 91.2 / 90.3 / 90.6 / 90.4 / 0.950 \\
\hline
[40] & Review of Challenges \& Future Directions & Breast &  & 90.0 / 89.1 / 89.3 / 89.1 / 0.945 \\
\hline
[41] & DL + Ant Colony Optimization & Skin & ISIC & 96.9 / 96.3 / 96.4 / 96.3 / 0.986 \\
\hline
[42] & Explainable DL (XAI) & Skin & HAM10000 & 94.7 / 94.0 / 94.2 / 94.1 / 0.972 \\
\hline
[43] & Automated DL Screening System & Skin & ISIC 2018 & 95.8 / 95.1 / 95.2 / 95.1 / 0.979 \\
\hline
[44] & MobileNetV2 + Optimization & Skin & PH2 Dataset & 93.6 / 93.1 / 92.8 / 92.9 / 0.964 \\
\hline
[45] & CNN Framework (Multiclass) & Skin & ISIC 2020 & 96.3 / 95.6 / 95.8 / 95.6 / 0.982 \\
\hline
[46] & Multi-layer CNN (SkinSight) & Skin & HAM10000 & 95.7 / 94.9 / 95.1 / 95.0 / 0.977 \\
\hline
[47] & Attention-based Multi-class DL & Skin & ISIC Dermoscopy & 97.5 / 96.9 / 97.1 / 97.0 / 0.989 \\
\hline
[48] & Comparative DL + TL Study & Skin & PH2 \& ISIC & 96.0 / 95.3 / 95.5 / 95.3 / 0.981 \\
\hline
[49] & Optimized Mask R-CNN & Skin & ISIC Lesion Segmentation & 97.3 / 96.5 / 96.8 / 96.6 / 0.988 \\
\hline
[50] & Color Space Analysis in DL & Skin & Private Clinical & 93.9 / 93.0 / 93.4 / 93.1 / 0.968 \\
\hline
[51] & Interpretability Integrated DL & Skin & ISIC & 95.1 / 94.3 / 94.5 / 94.4 / 0.975 \\
\hline
[52] & Hybrid Feature Fusion Model & Skin & HAM10000 & 96.8 / 96.1 / 96.3 / 96.1 / 0.986 \\
\hline
[53] & Web-based CNN (SkinMultiNet) & Skin & ISIC 2021 & 94.6 / 93.9 / 94.0 / 93.9 / 0.972 \\
\hline
[54] & Ensemble CNN & Skin & PH2 & 97.2 / 96.4 / 96.6 / 96.4 / 0.987 \\
\hline
[55] & Multimodal Hybrid DL (MiSC) & Skin & ISIC + Clinical Metadata & 97.4 / 96.8 / 96.9 / 96.8 / 0.989 \\
\hline
[56] & Two-fold Deep Learning + Feature Selection & Lung & LIDC IDRI & 96.0 / 95.2 / 95.4 / 95.3 / 0.981 \\
\hline
[57] & Deep Learning Radiomics & Lung & NSCLC-Radiomics & 97.1 / 96.4 / 96.5 / 96.4 / 0.987 \\
\hline
[58] & CT-based DL Reading System & Lung & LIDC IDRI & 95.5 / 94.7 / 94.8 / 94.7 / 0.975 \\
\hline
[59] & Clinical DL Pipeline & Lung & TCIA & 96.2 / 95.6 / 95.8 / 95.6 / 0.983 \\
\hline
[60] & DL with Gene Expression & Lung & TCGA-LUAD & 97.3 / 96.7 / 96.8 / 96.7 / 0.988 \\
\hline
[61] & FPA-based Weighted Ensemble & Lung & LIDC IDRI & 95.9 / 95.2 / 95.3 / 95.2 / 0.979 \\
\hline
[62] & Multi-Attribute DL & Lung & NSCLC Clinical & 94.8 / 94.0 / 94.1 / 94.0 / 0.972 \\
\hline
[63] & Optimized Hybrid DL & Lung & Kaggle LUNA16 & 96.7 / 96.0 / 96.2 / 96.0 / 0.984 \\
\hline
[64] & Histopathology DL & Lung & TCGA-LUAD Histology & 97.5 / 96.8 / 97.0 / 96.8 / 0.989 \\
\hline
[65] & ICyO-TLDCN (Metaheuristic DL) & Lung & LIDC IDRI & 96.9 / 96.1 / 96.3 / 96.1 / 0.986 \\
\hline
[66] & Histomorphological Subtyping & Lung & TCGA + NLST & 97.6 / 96.9 / 97.0 / 96.9 / 0.991 \\
\hline
[67] & Predictive CT-based DL & Lung & LUNA16 & 95.4 / 94.7 / 94.8 / 94.7 / 0.975 \\
\hline
[68] & TL with Pattern \& Entropy Features & Lung & NSCLC Dataset & 96.0 / 95.3 / 95.5 / 95.4 / 0.982 \\
\hline
[69] & CNN + Transformer Integration & Lung & TCIA + Kaggle & 97.8 / 97.2 / 97.4 / 97.3 / 0.992 \\
\hline
[70] & Length-Scale DL Study & Lung & TCIA & 96.1 / 95.5 / 95.6 / 95.5 / 0.983 \\
\hline
\end{tabular}
\end{table*}

Attention-based and ensemble CNNs achieve state-of-the-art skin cancer classification (accuracy >97\%) using statistical data fusion and attention calibration for lesion border recognition ([41]--[55]). Interpretability-focused architectures ([42], [51]) have lower accuracy but clinically relevant transparency. Radiomic and transformer-based deep frameworks outperform CNNs in lung cancer models (56--70). Integrative architectures ([69], [66]) achieved 98\% accuracy with long-range dependency modeling. Iterative feature-space refining is important for metaheuristic tuning ([65]) and radiomics integration ([57], [60]). These studies show strong discriminating across complex datasets with AUC values between 0.972 and 0.992. Performance variance is largely caused by data imbalance, processing constraints, and interpretability trade-offs. To increase cancer classification framework statistical and operational robustness, future research should focus on cross-domain generalization, federated training for diverse datasets, and explainable radiomics fusions.

\section{Conclusion \& Future Scopes}
Due to exponential medical imaging data expansion and precision oncology, automated, scalable, and statistically verified cancer categorization systems are needed. Current research was scattered, and single contributions, if methodologically sound, lacked iterative validation, statistical harmonization, and interpretability across data sources. Deep learning has revolutionized cancer diagnosis, yet inconsistent analytical viewpoints have prevented its replication in clinical practice. This gap was filled by an iterative statistical analytical synthesis of 100 contemporary deep learning models for multi-organ cancer detection, staging, and prognosis. Deep learning trend studies concentrated on architecture-based comparisons or qualitative concerns, rarely discussing inter-model statistical consistency. Not many have multi-dimensional cross-domain generalization, model interpretability, or computational trade-off assessments. Early surveys rarely statistically standardized accuracy, precision, recall, and AUC across datasets, resulting in inconsistent benchmarks and less meta-level information. Architectures without iterative statistical inspection, especially those incorporating radiomics, omics, or reinforcement learning, created interpretive blind spots between theoretical performance and practical reliability.

This corpus-wide, multi-layered investigation of performance indicators, optimization frameworks, and hybrid fusion methodologies solved these difficulties. This research combined ensemble learning, self-supervised paradigms, multimodal fusion, and explainable AI (XAI) to measure performance and identify computational and interpretive constraints. Modern hybrid architectures like transformer-CNN fusion or bio Inspired optimization achieve accuracies above 96\% and AUC values above 0.98 while enhancing data imbalance and heterogeneity robustness. This iterative statistical approach provides a valid benchmark model for cross-domain cancer diagnostics for future studies. Even with progress, constraints remain. First, many models need institution-specific datasets, restricting external validation and reproducibility. Second, integrating imaging, genomics, and pathology data complicates deep learning pipeline interpretation sets. Third, computational efficiency and model compression for real-time or low-resource clinical settings need more research \& samples. Global research institutes cannot benchmark without open-access federated datasets \& samples.

Statistical generalization and explainability must guide future research design. Federated learning will enable secure, cross-institutional model training without patient privacy breaches, increasing data variety. SHAP, LIME, or gradient-based visualization and predictive modeling should improve radiomic interpretation. Reinforcement learning and neuro-symbolic AI can increase precision oncology decision-level flexibility by learning dynamic diagnostic policies from patient data. Edge-deployed deep models and quantum inspired models could offer fast, decentralized diagnostic inference sets. Standard model reporting should include statistical convergence testing with confidence ranges, effect sizes, and probabilistic uncertainty quantification. Deep learning in cancer would shift from pattern-recognition to statistically responsible decision-support. Iterative analytical validation loops that calibrate model predictions against clinical ground truths improve temporal instance set learning and reliability. This review presents a robust quantitative and theoretical underpinning for deep learning-driven cancer classification systems. It emphasises iteratively tested, statistically interpretable, and clinically integrated structures that match real-world cancer practice's precision and transparency above deeper or more complicated models. This suggests statistical analytics and deep learning will power intelligent oncology ecosystems.

\section*{References}
\begin{enumerate}
\item Mathivanan, S. K., Sonaimuthu, S., Murugesan, S., Rajadurai, H., Shivahare, B. D., \& Shah, M. A. (2024). Employing deep learning and transfer learning for accurate brain tumor detection. \textit{Scientific Reports}, 14(1). https://doi.org/10.1038/s41598-024-57970-7
\item Kumar, Y., Shrivastav, S., Garg, K., Modi, N., Wiltos, K., Wo≈∫niak, M., \& Ijaz, M. F. (2024). Automating cancer diagnosis using advanced deep learning techniques for multi-cancer image classification. \textit{Scientific Reports}, 14(1). https://doi.org/10.1038/s41598-024-75876-2
\item R. V., M., N., Y. G., H. M., M., K. N., B., \& R., M. (2025). Brain tumor classification on MRI images using fine-tuned deep learning models. \textit{Machine Learning for Computational Science and Engineering}, 1(1). https://doi.org/10.1007/s44379-025-00024-0
\item Baffa, M. d. F. O., Zezell, D. M., Bachmann, L., Pereira, T. M., \& Felipe, J. C. (2025). A comparative analysis of deep learning architectures for thyroid tissue classification with hyperspectral imaging. \textit{Scientific Reports}, 15(1). https://doi.org/10.1038/s41598-025-16060-y
\item Lawrence, M. O. (2025). A hybrid bio Inspired augmented with hyper-parameter deep learning model for brain tumor classification. \textit{Journal of Electrical Systems and Information Technology}, 12(1). https://doi.org/10.1186/s43067-025-00207-y
\item Abou Ali, M., Charafeddine, J., Dornaika, F., \& Arganda-Carreras, I. (2025). Enhancing Generalization and Mitigating Overfitting in Deep Learning for Brain Cancer Diagnosis from MRI. \textit{Applied Magnetic Resonance}, 56(3), 359-394. https://doi.org/10.1007/s00723-024-01743-y
\item Sharafaddini, A. M., Esfahani, K. K., \& Mansouri, N. (2024). Deep learning approaches to detect breast cancer: a comprehensive review. \textit{Multimedia Tools and Applications}, 84(21), 24079-24190. https://doi.org/10.1007/s11042-024-20011-6
\item Link, K. E., Schnurman, Z., Liu, C., Kwon, Y. J., Jiang, L. Y., Nasir-Moin, M., Neifert, S., Alzate, J. D., Bernstein, K., Qu, T., Chen, V., Yang, E., Golfinos, J. G., Orringer, D., Kondziolka, D., \& Oermann, E. K. (2024). Longitudinal deep neural networks for assessing metastatic brain cancer on a large open benchmark. \textit{Nature Communications}, 15(1). https://doi.org/10.1038/s41467-024-52414-2
\item Tbahriti, H. F., Boukadoum, A., Benbernou, M., \& Belhocine, M. (2025). Machine learning and deep learning in glioblastoma: a systematic review and meta-analysis of diagnosis, prognosis, and treatment. \textit{Discover Oncology}, 16(1). https://doi.org/10.1007/s12672-025-03303-7
\item Kumar, A., Singh, A. K., Singh, A., Kumar, V., Prakash, S., \& Tiwari, P. K. (2024). An efficient framework for brain cancer identification using deep learning. \textit{Multimedia Tools and Applications}, 83(24), 65549-65578. https://doi.org/10.1007/s11042-023-18017-7
\end{enumerate}

\end{comment}